docs/aegis/roadmap/core-tooling-compliance.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Core Tooling Compliance
+area: Compliance
+status: In Progress
+progress: 20
+funding_required: false
+sponsors:
+  - Ericsson
+  # TODO: Can we disclose the other sponsor?
+index: 4
+previous:
+  url: gleam-openchain-compliance
+  title: Gleam OpenChain Compliance
+next:
+  url: core-tooling-governance-audit
+  title: Core Tooling Governance Audit
+---
+
+## Goal
+
+Implement Compliance Frameworks for core tooling
+
+## Impact
+
+By establishing a robust compliance framework for core tooling used in projects
+like Erlang, Elixir, Gleam, and Hex, this milestone will drive the consistent
+adoption of industry-leading security and quality standards across the open
+source ecosystem. By integrating OpenSSF Best Practices Badge, Security
+Insights, and ScoreCard, developers can more easily identify, measure, and
+address critical security gaps, thus ensuring that each project aligns with
+recognized best practices. The addition of a statistics tool to track adoption
+further amplifies transparency, enabling real-time monitoring of improvements
+and facilitating continuous growth in community-wide security awareness.
+Ultimately, this milestone fosters a healthier and more resilient open source
+environment, elevating trust for developers and end-users alike.
+
+## Deliverables
+
+* Projects:
+  - erlang
+  - elixir
+  - gleam
+  - hex
+* Implement OpenSSF Best Practices Badge
```

Comment:
the most basic one is ok, the next level requires 80% or more of code coverage.
I guess this is left open on purpose.

Reply:
That's why I left it open. Passing is simple. But silver and gold may have requirements that might be hard to implement.